app/models/abuse_report.rb

```diff
@@ -124,8 +124,9 @@ def send_report
   end
 
   def attach_work_download(ticket_id)
+    comment_id = url[%r{/comments/(\d*)}, 1]
     work_id = url[%r{/works/(\d+)}, 1]
-    return unless work_id
+    return unless work_id and not comment_id
```

Comment:
In addition to the Rubocop change, I think the condition would read a little more easily with the `.blank?` method – IMO `not` can get a little bit unwieldy with `unless`
```suggestion
    return unless work_id && comment_id.blank?
```

Reply:
`.blank?` wouldn't work here, but I changed it to avoid the `not`, is that better?